src/internal/task/cmds/cmds.go

```diff
@@ -0,0 +1,61 @@
+package cmds
+
+import (
+	"os"
+
+	pachdclient "github.com/pachyderm/pachyderm/v2/src/client"
+	"github.com/pachyderm/pachyderm/v2/src/internal/cmdutil"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+
+	"github.com/spf13/cobra"
+)
+
+func Cmds() []*cobra.Command {
+	var commands []*cobra.Command
+
+	var raw bool
+	var output string
+	outputFlags := cmdutil.OutputFlags(&raw, &output)
+
+	var group string
+	listTask := &cobra.Command{
+		Hidden: true, // don't show in the list of commands
+		Use:    "{{alias}} <service> [<namespace>]",
+		Short:  "Return info about tasks in a namespace.",
```

Comment:
Could we just remove the namespace part or change it to service. The service seems to be the primary command, and the namespace & group are just filters.